### PR TITLE
Magento2 332 category bulk export

### PR DESCRIPTION
**Pull request by augsteyer**
## Description

Bulk Export + Category observer logic mods. Unit tests + integration test (not hooked up to automation)

## Type of change

- [ ] Bug fix
- [x] New feature
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I updated the CHANGELOG.md

### CODE REVIEW

File: src/Console/Command/ExportCategories.php [hunk 1]
```diff
@@ -0,0 +1,205 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
```
Comment (andre-kraus) [inline src/Console/Command/ExportCategories.php]: Hm, wrong license header I guess.

File: src/Observer/Catalog/Category/Delete.php [hunk 2]
```diff
@@ -60,9 +54,6 @@ public function __construct(Config $config, Category $category, ManagerInterface
      */
     public function execute(Observer $observer): void
     {
-        if (!$this->config->isEventEnabled()) {
```
Comment (andre-kraus) [inline src/Observer/Catalog/Category/Delete.php]: Why removing this (also for the delete observer only)?

Comment (augsteyer) [reply src/Observer/Catalog/Category/Delete.php]: already present inside delete method. Could not do the same for others as create call has special logic that checks all store views of a store group. Delete and update just care about current context.

**Comment by augsteyer** [comment]
can ignore pr fail, this is travis related. Seems like it is running tests on master branch of this repo, which means it's loading an old sdk and not finding classes. Will need to adjust when hooking up integration tests to travis.

**Pull request merged**